packages/cli/src/ceramic-daemon.ts

```diff
@@ -212,12 +215,12 @@ class CeramicDaemon {
     const stats = `maxHealthyCpu=${this.maxHealthyCpu} cpuUsage=${cpuUsage} freeCpu=${freeCpu} maxHealthyMemory=${this.maxHealthyMemory} memoryUsage=${memUsage} freeMemory=${freeMemory}`
 
     if (cpuUsage > this.maxHealthyCpu || memUsage > this.maxHealthyMemory) {
-      this.logger.error('Ceramic failed a healthcheck')
-      res.status(503).send('Insufficient resources')
+      this.logger.debug(stats)
```

Comment:
hmm... it still feels useful to log an error if we actually fail a healthcheck I think?  I think this should probably still be logger.error, with a message that we failed a healthcheck, *and* with the stats included.

Reply:
Other comment addresses this. so yes the message will get logged as an error and the stats will get logged as debug.